client/public/index.html

```diff
@@ -23,5 +23,11 @@
   </head>
   <body>
     <div id="root"></div>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/2.3.0/socket.io.js"></script>
+        <script>
+          document.addEventListener("DOMContentLoaded", function() {
+          var socket = io.connect("http://localhost:5000/")
```

Comment:
ahh we can't do this like that, it wouldn't work on Netlify

Reply:
why wouldnt it? how should it be?